docs/integrations/amazon-aws/ses.md

```diff
@@ -312,11 +312,11 @@ See the details of various SES CloudTrail events including the identity, get sen
 **Email Address Event Details**. See the details of the email address event in the last seven days including the event time, event name, event type, type, user, AWS region, source IP address, event status, account ID, request ID, error code, error message, and event count, displayed in a table.
 
 
-#### Notification Overview
+### Notification Overview
 
 See the overview of SES notifications including the source IP locations, notification types, mail source, and notification type.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/Overview.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/SES-Notification.png')} alt="AWS SES Notification" />
 
 **Mail Source IP Locations**. See the count and location of the mail source IP addresses in the last 24 hours on a world map.
```

Comment:
```suggestion
**Mail Source IP locations**. See the count and location of the mail source IP addresses in the last 24 hours on a world map.
```

Reply:
<img width="604" alt="Screenshot 2023-06-09 at 4 21 31 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/88306913/60512d48-cd35-4916-b79b-54896cbcf653">
Capital 'L' is expected here as it is a defined panel name.